src/visualization/visualize_data.jl

```diff
@@ -655,15 +656,15 @@ function plot_grid_dispatched(results::Dict;seed=1111,quant::Float64=0.5,size_fi
 end
 
 """
-    HydroPowerModels.plot_aggregated_results(results::Dict)
+    HydroPowerModels.plot_aggregated_results(results::Dict;nc::Int=3)
```

Comment:
You have to change the doc
Parameter:
- result ...
- nc

Also, nc is a bad name